iina/AutoFileMatcher.swift

```diff
@@ -45,9 +46,11 @@ class AutoFileMatcher {
 
     // group by extension
     for file in files {
-      let fileInfo = FileInfo(file)
-      if let mediaType = Utility.mediaType(forExtension: fileInfo.ext) {
-        filesGroupedByMediaType[mediaType]!.append(fileInfo)
+      autoreleasepool{
```

Comment:
`autoreleasepool` is useful when you're working with `autorelease`d types. You're working with Swift `struct`s here.